psc-package/Main.hs

```diff
@@ -26,6 +26,7 @@ import qualified Options.Applicative as Opts
 import qualified Paths_purescript as Paths
 import qualified System.IO as IO
 import           Turtle hiding (fold)
+import 			 Control.Lens((<.), toListOf, to, ifolded, withIndex)
```

Comment:
These tabs should be spaces. Usually there is a space between the module name and the open parenthesis too. :)